.github/workflows/validate-docs.yml

```
@@ -0,0 +1,212 @@
+            tools/docs-manager/node_modules
+          key: ${{ runner.os }}-bun-docs-${{ hashFiles('bun.lockb', 'package.json', 'tools/docs-manager/package.json') }}
+          restore-keys: |
+            ${{ runner.os }}-bun-docs-
+
+      - name: Setup retry function
+        timeout-minutes: 1
+        run: |
+          # Create retry function as a system command
+          sudo tee /usr/local/bin/retry_command << 'EOF'
+          #!/bin/bash
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)